.circleci/config.yml

```diff
@@ -13,6 +13,12 @@ jobs:
       # CircleCI maintains a library of pre-built images
```

Comment:
if I could just see files that are different than last week's that would be ideal